.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-    "liveServer.settings.port": 5502
+    "liveServer.settings.port": 5503
 }
```

Comment:
It is important that files/folders not related to the project are not committed to the project repo. Kindly remove the `.vscode` directory and add it to the `.gitignore` file so as to avoid committing such a folder in the future.